lib/modules/device/DevicesController.ts

```diff
@@ -64,9 +64,7 @@ export class DevicesController {
         },
         upsert: {
           handler: this.upsert.bind(this),
-          http: [
-            { path: "device-manager/:engineId/devices/:_id", verb: "post" },
-          ],
+          http: [{ path: "device-manager/:engineId/devices", verb: "put" }],
```

Comment:
This is a breaking API change, this should be marked as such in the commits and lead to a semver-major release.